internal/edgedbtypes/datetime_test.go

```diff
@@ -106,6 +106,56 @@ func TestNewLocalDate(t *testing.T) {
 	}
 }
 
+func TestLocalTimeString(t *testing.T) {
+	samples := []struct {
+		str string
+		d   LocalTime
+	}{
+		{"00:00:00", LocalTime{0}},
+		{"00:00:00.000001", LocalTime{1}},
+		{"00:00:00.00001", LocalTime{10}},
+		{"00:00:00.0001", LocalTime{100}},
+		{"00:00:00.001", LocalTime{1000}},
+		{"00:00:00.01", LocalTime{10000}},
+		{"00:00:00.1", LocalTime{100000}},
+		{"00:00:00.123456", LocalTime{123456}},
+		{"05:04:03", LocalTime{18_243_000_000}},
+		{"20:39:57", LocalTime{74_397_000_000}},
+		{"23:59:59.999999", LocalTime{86_399_999_999}},
```

Comment:
Can you add a test that it's impossible to create local time with 25 hours, or 99 minutes, etc

Reply:
added :heavy_check_mark: